src/actions/index.js

```diff
@@ -1,25 +1,49 @@
+import { geocodeGoogle, breezoMeter } from '../api';
+
 export const RECEIVE_LOCATION = 'RECEIVE_LOCATION';
 
-export function receiveLocation(location, json) {
+export function receiveLocation(json) {
   //dispatching an action to the store
```

Comment:
you should use object destructuring here to pull out the properties from `json` that you need. Then use shorthand object assignment: `return { date, name, aqi, etc... }`.
 https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment